qa-tests-backend/src/test/groovy/UpgradesTest.groovy

```diff
@@ -22,15 +22,9 @@ class UpgradesTest extends BaseSpecification {
     private final static String POLICIES_JSON_PATH =
             Env.get("POLICIES_JSON_RELATIVE_PATH", "../pkg/defaults/policies/files")
 
-    private static final String VULNERABILITY_RESOURCE_TYPE =
-        isPostgresRun() ?
-            "nodeVulnerabilities" :
-            "vulnerabilities"
-
-    private static final String COMPONENT_RESOURCE_TYPE =
-        isPostgresRun() ?
-            "nodeComponents" :
-            "components"
+    private static final String VULNERABILITY_RESOURCE_TYPE = "nodeVulnerabilities"
```

Comment:
There is still a condition in the `tests/upgrade/legacy_to_postgres_run.sh` that uses this.  That test currently tests that we can upgrade from 3.74 -> 4.4 and back to 3.74.  This test executes on the rollback to 3.74 to verify the rollback was successful.  I contend we should keep that part of the upgrade test until 4.4 has been out of support for a bit to ensure patches do not break that flow.  So this one needs to stay or we need to figure out another way to satisfy that case.

Reply:
Ah, great point - reverted the changes for the upgrade tests.